TAO/TAO_IDL/docs/annotations.md

```diff
@@ -345,11 +375,11 @@ these lines would also need to be added:
   if (document)
     {
       const char *comment =
-        AST_Annotation_Member::narrow_from_decl ((*document)["comment"])->
+        dynamic_cast<AST_Annotation_Member *> ((*document)["comment"])->
```

Comment:
Out of curiosity, why do you use dynamic_cast instead of narrow_from_decl?

Reply:
`narrow_from_decl` is defined in `include/idl_narrow.h` where it's just a `dyanmic_cast`. It's a left over from an old narrowing system from the original pre C++98 compiler when they didn't apparently have `dynamic_cast`.

Reply:
I guess I should also say that we should prefer standard C++ over pre-standard stuff whenever it's possible to.

Reply:
Maybe make a todo to make this a global change in TAO_IDL to simplify the code overall